transmogrifier/sources/json/aardvark.py

```diff
@@ -309,14 +304,22 @@ def get_locations(
         for aardvark_location_field, kind_value in aardvark_location_fields.items():
             if aardvark_location_field not in source_record:
                 continue
-            try:
+            if (
+                geodata_string := source_record[aardvark_location_field]
+            ) and "ENVELOPE" in source_record[aardvark_location_field]:
+                locations.append(
+                    timdex.Location(
+                        geoshape=geodata_string.replace("ENVELOPE", "BBOX"),
+                        kind=kind_value,
+                    )
+                )
+            else:
                 message = (
-                    f"Geometry field '{aardvark_location_field}' found, but "
-                    f"currently not mapped."
+                    f"Record ID '{source_record_id}': "
+                    f"Unable to parse geodata string '{geodata_string}' "
+                    f"in '{aardvark_location_field}'"
```

Comment:
Sat on this for awhile, but I think it looks good.  Maybe we'll get into more WKT string types going forward, but for now I think we can be relatively confident they are all `ENVELOPE(...)` strings.  And, therefore, if those fields are present and _don't_ have that `ENVELOPE` string, then it feels accurate we cannot "parse" it.